### PR TITLE
docs(): page link improvement part one 

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Remove warning `Dangling Links Found`

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
This PR, Remove warning `Dangling Links Found`(see screenshot)

I split this improvement into 3 PRs to do a little faster review.

screenshot

### CODE REVIEW

File: content/cli/usages.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ $ nest g <schematic> <name> [options]
 | `pipe`        | `pi`  | Generate a pipe declaration.                                                                        |
 | `provider`    | `pr`  | Generate a provider declaration.                                                                    |
 | `resolver`    | `r`   | Generate a resolver declaration.
-| `resource`    | `res` | Generate a new CRUD resource. See the [CRUD (resource) generator](/recipes/crud-generator) for more details.                                                                    |
+| `resource`    | `res` | Generate a new CRUD resource. See the [CRUD (resource) generator](https://docs.nestjs.com/recipes/crud-generator) for more details.                                                                    |
```
Comment (kamilmysliwiec) [inline content/cli/usages.md:73]: What's the reason for this change?

Comment (kamilmysliwiec) [reply content/cli/usages.md:73]: (this question applies to most of the updates in this PR)

Comment (Tony133) [reply content/cli/usages.md:73]: I did these PRs, to remove the warnings that occur when we do `npm run build` (see screenshot)

To remove them I added this part `https://docs.nestjs.com`to each url where there was the warning.
It should always be added to have no warnings in the future.

screenshot

Comment (kamilmysliwiec) [reply content/cli/usages.md:73]: The local version of the application won't be working as expected now as every link will route you to the deployed version of the app (instead of the local one running on your machine) making the development process quite frustrating

Comment (Tony133) [reply content/cli/usages.md:73]: Got it, can this be fixed when a new version of NestJS (v9) comes out by updating the site?

Comment (kamilmysliwiec) [reply content/cli/usages.md:73]: Not sure if I'm following(?). Explicitly adding `docs.nestjs.com` to the link just makes no sense as the application won't run as expected between different environments.

Comment (Tony133) [reply content/cli/usages.md:73]: Understood , In many URLs on the site this is already there (see screenshot) but in others it is missing and generates the warning.

screensho2

Comment (Tony133) [reply content/cli/usages.md:73]: These are the same warnings that you also find in the Netfily console (see screenshot)

screenshot3

Comment (kamilmysliwiec) [reply content/cli/usages.md:73]: > Understood, In many URLs on the site this is already there (see screenshot) but in others it is missing and generates the warning.

We use `docs.nestjs.com` explicitly **only** if we want a link to be opened in a new tab

Comment (Tony133) [reply content/cli/usages.md:73]: Ok, so we leave the warnings like this? 
You can also find them in the Netfily console (you saw the screenshot above)? 

However it would be bad for each link to open a new tab, it is true that it depends on the cases, but for a guide it would be interesting, so you can pick up where you were reading.

Comment (kamilmysliwiec) [reply content/cli/usages.md:73]: > Ok, so we leave the warnings like this?

yeah these warnings are acceptable ✅

Comment (Tony133) [reply content/cli/usages.md:73]: Ok 👍, if you want these links to open in another tab ok, otherwise close these PRs as well. Then if you want to think about it let me know, no problem for me. 😉 👍
